universal-application-tool-0.0.1/app/services/question/QuestionDefinition.java

```diff
@@ -109,6 +108,19 @@ public QuestionType getQuestionType() {
     return ImmutableMap.of("text", ScalarType.STRING);
   }
 
+  /** Get a map of scalars stored by this question definition. */
+  @JsonIgnore
+  public ImmutableMap<String, ScalarType> getScalars(boolean includeFullPath) {
```

Comment:
Issue (non-blocking): avoid using boolean params in public methods

see https://engdoc.corp.google.com/eng/doc/devguide/java/practices/boolean-parameters.md?cl=head